docgen/src/examples.md

```diff
@@ -1,11 +1,10 @@
 ---
 title: Examples
-layout: examples.pug
+layout: main-entry.pug
 category: examples
+tocVisibility: false
 ---
-# Examples
 
-* e-commerce: [live demo](examples/e-commerce/index.html) [code](https://github.com/algolia/instantsearch.js/tree/v2/docgen/src/examples/e-commerce)
-* tourism: [live demo](examples/tourism/index.html) [code](https://github.com/algolia/instantsearch.js/tree/v2/docgen/src/examples/tourism)
-* media: [live demo](examples/media/index.html) [code](https://github.com/algolia/instantsearch.js/tree/v2/docgen/src/examples/media)
-* Material-UI: [live demo](examples/material-ui/index.html) [code](https://github.com/algolia/instantsearch.js/tree/v2/docgen/src/examples/material-ui)
+To demonstrate the flexibility of react-instantsearch we have created some demos.
```

Comment:
Do we plan to have this page displaying the examples more like the IS.js doc? (with images and re-adding source code links)